osidb_bindings/bindings/README.md

```diff
@@ -25,9 +25,10 @@ from python_client.models import MyDataModel
 from python_client.api.my_tag import get_my_data_model
 from python_client.types import Response
 
-my_data: MyDataModel = get_my_data_model.sync(client=client)
-# or if you need more info (e.g. status_code)
-response: Response[MyDataModel] = get_my_data_model.sync_detailed(client=client)
+with client as client:
```

Comment:
Also here, why the `as`?

Reply:
This README is generated by the openapi-python-client, there is a new functionality where the client works like a context manager, see the template in `templates_0.22.0`, but I decided to not to adopt this new functionality yet and keep the old one. This README is just for the lower level client which we build the abstraction on.